tests/foreman/cli/test_setting.py

```diff
@@ -233,10 +246,15 @@ def test_negative_update_email_reply_address(self):
 
         :caseimportance: low
 
-        :caseautomation: notautomated
+        :caseautomation: automated
         """
+        with self.assertRaises(CLIReturnCodeError):
+            email_reply_address_value = invalid_emails_list()
```

Comment:
shouldn't you use similar loop construct to one you used in previous test case: `for email in invalid_emails_list():`?

Reply:
thanks @oshtaier , I will change it